pontoon/base/static/js/translate.js

```diff
@@ -706,11 +706,11 @@ var Pontoon = (function (my) {
       }
 
       var before = this.cachedTranslation,
-          after = $('#translation').val();
+          after = this.fluent.serializeTranslation(entity, $('#translation').val());
```

Comment:
In most places you've replaced `$('#translation')` with `$('#editor textarea:visible:first')`, why not here?

Reply:
Here, we compare the old (`before`) and the current (`after`) state of the string by comparing the source of the string, which is stored in `#translation`.